openhabian-setup.sh

```diff
@@ -124,6 +124,7 @@ if [[ -n "$UNATTENDED" ]]; then
   jsscripting_npm_install "openhab_rules_tools"
   zram_setup
   exim_setup
+  nut_setup
   permissions_corrections
```

Comment:
setup without prior install ?

Reply:
I chose the same pattern as for the exim setup: the `nut_setup` methods installs nut if not already installed.